packages/mdc-drawer/_mixins.scss

```diff
@@ -119,13 +110,11 @@
 }
 
 @mixin mdc-drawer-surface-fill-color($color) {
-  $value: mdc-theme-prop-value($color);
-
-  @include mdc-theme-prop(background-color, $value);
+  @include mdc-side-sheet-surface-fill-color($color);
 }
 
 @mixin mdc-drawer-scrim-fill-color($color) {
-  $value: rgba(mdc-theme-prop-value($color), $mdc-drawer-modal-scrim-opacity);
+  $value: rgba(mdc-theme-prop-value($color), $mdc-side-sheet-modal-scrim-opacity);
```

Comment:
So, are we not sharing `mdc-side-sheet-scrim` with drawer?

Reply:
I guess we can do that provided no one needs to customize the scrim colors for each independently (which should probably be discouraged anyway). I'll see what that ends up looking like.

Reply:
On second thought, I think these need to be separate, otherwise you're going to run into issues if you have a dismissible drawer but a modal side sheet, or vice versa.

Reply:
I was wondering about app-content too.